src/JXGBoard.js

```diff
@@ -10,17 +13,30 @@ export default class JXGBoard extends React.Component {
     }
 
     renderJXBoard(options) {
-        // The jsxgraph npm package is out of date so I'm including
-        // this package globally for now.
-        if (!window.JXG) {
+        // Don't render JSXGraph in jest. This should be possible since I'm
+        // using jsdom, but jsxgraph can't find the element when I try and
+        // initialize it like this:
+        //
+        // const div = document.createElement('div');
+        // div.setAttribute('id', 'id-test');
+        // ReactDOM.render(
+        //     <JXGBoard
+        //          id={'id-test'}
+        //          gType={0}
+        //          gShowIntersection={true} />,
+        //     div);
```

Comment:
Hmmm, actually I'm realizing why this might not be working. I'm rendering the react element *inside* the div that has the 'id-test' id. It slipped my mind that this div is used as the container (second argument to ReactDOM.render()). It's likely that creating a child div within here with the id will solve my jest problem.

So, this can be merged fine as is. There's more important work to be done at the moment. But this is a note to myself to make that and see if it works at some point.